### PR TITLE
[mempool] BlockGasLimit is now configurable

### PR DESCRIPTION
**Pull request by fosskers**

### CODE REVIEW

File: src/Chainweb/Chainweb.hs [hunk 1]
```diff
@@ -230,6 +232,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configThrottleRate = 1000
     , _configMempoolP2p = defaultEnableConfig defaultMempoolP2pConfig
     , _configPruneChainDatabase = True
+    , _configBlockGasLimit = 100000
```
Comment (fosskers) [inline src/Chainweb/Chainweb.hs]: Is this default still appropriate, given that gas values went up for many primitives?

Comment (gregorycollins) [reply src/Chainweb/Chainweb.hs]: We need to set it so a full block takes less than a half second to process, that's the target I think.

**Comment by fosskers** [comment]
Shall we address that in a follow-up, once we have more data?

**Comment by gregorycollins** [comment]
> Shall we address that in a follow-up, once we have more experiment data?

Probably -- it would probably behoove us, now that most of the validations are in, to spend some time again on block validation performance before we adjust it

**Pull request merged**